### PR TITLE
Add datasource to Prometheus dashboard

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Adding a new datasource variable to the Prometheus dashboard. This will allow users to select the Prometheus datasource when viewing the dashboard.

### CODE REVIEW

File: helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json [hunk 1]
```diff
@@ -1730,6 +1730,24 @@
   ],
   "templating": {
     "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "Mimir",
```
Comment (QuentinBisson) [inline helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json:1736]: Can you set text and value to default instead?

File: helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json [hunk 2]
```diff
@@ -1742,7 +1760,7 @@
         },
         "datasource": {
```
Comment (QuentinBisson) [inline helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json:1761]: Thé whole thing:

"datasource": {
          "type": "prometheus",
          "uid": "${datasource}"
        },

Can bé replaced with "datasource": "$datasource", but it's not blocking

File: helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json (hunk 1 above)
Comment (QuentinBisson) [reply helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json:1736]: To align with existing values

File: helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json [hunk 3]
```diff
@@ -1730,6 +1730,24 @@
...
+        "current": {
+          "selected": false,
+          "text": "Mimir",
+          "value": "mimir"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
```
Comment (QuentinBisson) [inline helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json:1743]:
```suggestion
        "options": [],
        "label": "Data source",
```

**Review by QuentinBisson — APPROVED** [review]

File: helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json (hunk 2 above)
Comment (TheoBrigitte) [reply helm/dashboards/charts/public_dashboards/dashboards/shared/public/prometheus.json:1763]: I'm keeping this as is

**Pull request merged**